foundations/installations/installations.md

```diff
@@ -261,14 +256,82 @@ If you encounter an error requesting you to disable **Intel RST** while attempti
+- Enter the following command
+
+~~~powershell
+wsl --install
+~~~
+
+- After a few minutes you'll be prompted to reboot your computer; do so.
+- You should see an open Powershell window, prompting you to enter a username and a password. Your username should be lowercase, but can otherwise be whatever suits you. You'll also need to enter a new password.
+- When entering your password you might notice that you aren't seeing any visual feedback. This is a standard security feature in Linux, and will also happen in all future cases where you need to enter some form of password. Rest assured that you are actually inputting something.
+
+#### Step 1.3.1: Install Windows Terminal (Windows 10 only)
+
+Windows Terminal is an terminal application that lets you more easily customize and run terminals, as well as supporting multiple tabs which can each run their own different terminals.
+
+- [Install Window's Terminal](https://learn.microsoft.com/en-us/windows/terminal/install) by using the direct install option.
+
+#### Step 1.3.2: Setting WSL2 as default (Optional)
+
+Unless you are regularly using other terminals on your computer we recommend that you set WSL as the default terminal program when you open Windows Terminal.
+
+- Open Windows Terminal by searching for terminal in your applications.
+- Click the dropdown next to the new tab button (at the top of the windows), and select Settings.
+- You should see a Default Profile option with a dropdown next to it.
+- In the dropdown select Ubuntu
+- Click save at the bottom of the page
+
+### Step 2 Opening WSL2
+
+On Windows there are three primary ways to open WSL2.
+
+- If you set Windows Terminal to open a Ubuntu terminal by default you can start a new WSL2 session by simply opening the terminal app.
+- Otherwise you can open Windows Terminal, click the dropdown next to the new tab button (at the top of the windows), and select Ubuntu.
+- If you search for Ubuntu in the application search bar you should see a application titled Ubuntu; simply open it to start a new terminal session.
+
+<div class="lesson-note lesson-note--tip">
+You might notice than when you open WSL2 via Windows Terminal you'll see a window with a different color scheme and a different icon then when opening by searching for Ubuntu in your applications. This is because Windows Terminal comes with a default color scheme for Ubuntu meant to emulate how a real Ubuntu terminal looks. This difference is purely cosmetic, and there is no practical difference between the two/
+</div>
+
+<div class="lesson-note lesson-note--warning">
+When opening WSL2 ensure that you do not see /mnt/c at the start of the line. /mnt/c is where your Windows installation lives when working within WSL2, and messing around there can have unintended consequences
```

Comment:
```suggestion
When opening your WSL2 terminal ensure that you do not see /mnt/c at the start of the line. /mnt/c is where your Windows installation lives when working within WSL2, and messing around there can have unintended consequences
```

Possible clarification. Learners might not be fully comfortable with the phraseology, but by now in this lesson they are very familiar with "terminal" and are creating the connection of WSL2 living in there.